resources/assets/lib/chat/chat-state-store.ts

```diff
@@ -62,6 +58,14 @@ export default class ChatStateStore implements DispatchListener {
       }
     });
 
+    autorun(() => {
+      if (this.isChatMounted) {
+        document.querySelector('html')?.classList.add('u-chat');
+      } else {
+        document.querySelector('html')?.classList.remove('u-chat');
```

Comment:
unmount of the previous page happening after the current page mounts does cause the layout to flash when going through history

Reply:
maybe at least the scrollbar thing should just be done in the chat component class instead of global ಠ_ಠ

Reply:
or just move the `u-chat` to be part of `chat/index.blade.php`'s `bodyAdditionalClasses` if that works...

Reply:
as far as I know this was on `html` because of iOS...except it turns out there's new problems with with the layout on the latest iOS ಠ_ಠ